lib/fluent/plugin/out_http.rb

```diff
@@ -120,6 +122,11 @@ def set_text_body(req, data)
     req['Content-Type'] = 'text/plain'
   end
 
+  def set_raw_body(req, data)
+    req.body = data.to_s
```

Comment:
I've found that `req.body` should be String because `String#bytesize` will be called requesting phase in net/http.